config/locales/en.yml

```diff
@@ -595,8 +595,6 @@ en:
       over_soft_max_for_la_combined: "You told us the combined income of this household is %{combined_income}. This seems high.  Are you sure this is correct?"
     rent:
       outside_range_title: "You told us the rent is %{brent}"
-      min_hint_text: "The minimum rent expected for this type of property in this local authority is %{soft_min_for_period}."
-      max_hint_text: "The maximum rent expected for this type of property in this local authority is %{soft_max_for_period}."
```

Comment:
did you double check these aren't being used anywhere else, ie. global search soft_validations.rent.min_hint_text

Reply:
yes, however I want to check again now :)